docs/tableBeamGroup1.adoc

```diff
@@ -154,14 +156,22 @@ e|Variables | |
  |{var}float detected_bottom_range(ping_time, beam) |O |Range from the transducer face where the bottom detection criteria were encountered for the amplitude or the phase of the backscattered echoes. The range of the bottom at index bottom_index with a monostatic transducer and if a constant sound speed is applied is given by detected_bottom_range= sound_speed_at_transducer*(blanking_interval+bottom_index*sample_interval - sample_time_offset)/2.
  3+|{attr}:long_name = "Detected range of the bottom" 
  3+|{attr}:units = "m" 
- 3+|{attr}:valid_min = "0.0" 
+ 3+|{attr}:valid_min = "0.0"
+
+ |{var}float transducer_impedance(ping_time, subbeam) |MA |Impedance of the transducer. This is the impedance of the load over which the transceiver measures voltage on the transducer subbeam. Necessary for conversion equation type 4.
+ 3+|{attr}:long_name = "Impedance of transducer"
+ 3+|{attr}:units = "ohm"
+
+ |{var}float transceiver_impedance(ping_time, subbeam) |MA |Impedance of the transceiver. This is the impedance of the transducer subbeam. Necessary for conversion equation type 4.
+ 3+|{attr}:long_name = "Impedance of transceiver"
+ 3+|{attr}:units = "ohm"
  
  |{var}sample_t time_varied_gain(ping_time) |MA |Time-varied gain (TVG) used for each ping. Should contain TVG coefficient vectors. Necessary for type 2 conversion equations.
  3+|{attr}:long_name = "Time-varied-gain coefficients" 
  3+|{attr}:units = "dB" 
  3+|{attr}:coordinates = "ping_time platform_latitude platform_longitude" 
  
- |{var}float transducer_gain(ping_time, beam) |MA |Gain of the transducer beam. This is the parameter that is set from a calibration exercise. Necessary for conversion equation type 1.
+ |{var}float transducer_gain(ping_time, beam, frequency) |MA |Gain of the transducer beam. This is the parameter that is set from a calibration exercise. Necessary for conversion equation type 1, 3 and 4.
```

Comment:
I think there needs to be a coordinate variable for the frequency dimension.

Reply:
Agreed and added